testrunner/js/helpers.js

```diff
@@ -28,4 +28,40 @@
 function isFunction(functionToCheck) {
 	var getType = {};
 	return functionToCheck && getType.toString.call(functionToCheck) === '[object Function]';
+}
+
+/**
+ * Counts the number of keys in an object.
+*/
+function numberOfKeys(obj) {
```

Comment:
I think you can use Object.keys(obj).length for the same purpose, but this doesn't hurt/is more immediately readable